packages/frames.js/src/types.ts

```diff
@@ -37,7 +37,7 @@ export const frameErrorKeys = [
   "fc:frame:button:4",
   "og:image",
   "og:title",
-  "of:accepts",
+  "of:accepts", // @todo remove this as there is no validation in validateFrame function
```

Comment:
@davidfurlong this causes issue in `frame-debugger.tsx`, for now it is fixed by filtering out this key but what should we do about this?

Should we remove it since it isn't a part of `FrameFlattened` or should we somehow add it there?

The issue is that `of:accepts:${id}` is dynamic, so we can't type it upfront in `FrameFlattened` since `id` can be anything.